src/SingleProject/Resizetizer/src/Resizetizer.csproj

```diff
@@ -72,10 +72,15 @@
       <_CopyItems Include="@(_ResizetizerFiles)" />
     </ItemGroup>
     <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)%(_CopyItems.Arch)" ContinueOnError="true" Retries="0" />
+    <MakeDir Directories="$(IntermediateOutputPath)adjustments" />
```

Comment:
Is `adjustments` the right name? I guess it doesn't really matter too much, it doesn't end up in the final package, right?  Can anyone think of a bit more intuitive of a name?  `aggregate`?  Not a deal breaker as it  is now, just thinking out loud.

Reply:
it is just a folder in the obj - I am basically editing the one dll and need a temp location. I used adjustments to match the name of the task - but we can do anything.